os.go

```diff
@@ -34,3 +35,80 @@ func mkfs(dev, fstype string) error {
 	}
 	return nil
 }
+
+func umount(mntpoint string) error {
+	tmpDir := os.TempDir()
+	var stderr bytes.Buffer
+
+	cmd := exec.Command(
+		"/bin/umount",
+		tmpDir,
+	)
+	cmd.Stderr = &stderr
+	if err := cmd.Run(); err != nil {
+		logrus.Errorf("umount stderr: %s", stderr.String())
+		return err
+	}
+	return nil
+}
+
+func tempMount(dev, fstype string, mountOptions ...string) error {
+	tmpDir := os.TempDir()
+	var stderr bytes.Buffer
+
+	mountArgs := []string{
+		"-t",
+		fstype,
+	}
+	mountArgs = append(mountArgs, mountOptions...)
+	mountArgs = append(mountArgs,
+		dev,
+		tmpDir,
+	)
+	cmd := exec.Command(
+		"/bin/mount",
+		mountArgs...,
+	)
+	cmd.Stderr = &stderr
+	if err := cmd.Run(); err != nil {
+		logrus.Errorf("mount stderr: %s", stderr.String())
+		return err
+	}
+	return nil
+}
+
+func chown(dir string, uid uint32, gid uint32) error {
+	uidgid := fmt.Sprintf("%d:%d", uid, gid)
+	var stderr bytes.Buffer
+	cmd := exec.Command(
+		"/bin/chown",
+		"-R",
+		uidgid,
+		dir,
+	)
+	cmd.Stderr = &stderr
+	if err := cmd.Run(); err != nil {
+		logrus.Errorf("chown stderr: %s", stderr.String())
+		return err
+	}
+	return nil
+}
```

Comment:
I _think_ this can be replaced with `fs.WalkDir` + `os.Chown`? This would get rid of yet another `exec.Command` instance :+1:

Reply:
We can try this.

Tbh, I wonder whether we even need the fs.WalkDir then, as we are mounting an empty volume that at most contains a lost+found folder which I honestly think should be irrelevant anyways.

Reply:
ah yes, good point. It might even be a good place to _check_ if the folder is empty and warn if it isn't?